packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -625,7 +625,7 @@ const conf = (module.exports = convict({
       format: String,
     },
     managementTokenTTL: {
-      default: 900,
+      default: 1800,
```

Comment:
We *might* want to double check with ops to be sure that there is no `SUBSCRIPTIONS_MANAGEMENT_TOKEN_TTL` configured in any environment and that this default is used. (I'm pretty sure we're fine here - I don't think this got configured unless we asked for it to be done.)

Reply:
Just verified with @jrgm in Slack that the `SUBSCRIPTIONS_MANAGEMENT_TOKEN_TTL` isn't set via env var, ops just takes the defaults. Good catch, though :+1:

Reply:
Answered 6a68 in slack channel, but no stage/prod do not configure an override of the default SUBSCRIPTIONS_MANAGEMENT_TOKEN_TTL